webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric-grading.js

```diff
@@ -63,7 +63,15 @@ export class SakaiRubricGrading extends RubricsElement {
   render() {
 
     return html`
-      <h3 style="margin-bottom: 10px;">${this.rubric.title}</h3>
+      <h3 style="margin-bottom: 10px;">${this.rubric.title}
+	  	    ${true ? html`
```

Comment:
We need a property in the serverConfigurationService to hide this.

Could we use 'true &&' instead of 'true ? : '

Reply:
You don't need a template here at all. Just put the span straight in there. You're already inside the main render template.